test/api/users-primary-body.test.js

```diff
@@ -131,10 +131,10 @@ describe('User primary body setting', () => {
             body: { primary_body_id: null }
         });
 
-        expect(res.statusCode).toEqual(403);
-        expect(res.body.success).toEqual(false);
-        expect(res.body).not.toHaveProperty('data');
-        expect(res.body).toHaveProperty('message');
+        expect(res.statusCode).toEqual(200);
```

Comment:
Why are you changing this test? Was it wrong from the start or is this only for the member itself?

Reply:
yeah, it wasn't supposed to work like that and was wrong from the start (it expected for the backend to return 403, while the backend should return 200 in real life)